packages/shared/styles/styleUtils/modifiers/backgroundColor.ts

```diff
@@ -0,0 +1,7 @@
+import { css } from "react-emotion";
+
+export const background = color => {
+  return css`
+    background-color: ${color} !important;
```

Comment:
Do we really need the !important? It always sends shivers down my spine.

Reply:
I wanted to be consistent with existing modifiers, for example - https://github.com/dcos-labs/ui-kit/blob/90280672f59c9af244a8c2e5838294c3bb5969a7/packages/shared/styles/styleUtils/modifiers/display.ts#L19 .

Reply:
@Poltergeist @mperrotti Could you give me some background on this?

Reply:
I think it's safe to remove the `!important`s on any modifiers.
`display` modifiers were some of the first ones I wrote when I was new to using emotion, so I was using the brute-force approach I'd use in regular CSS to make sure modifier styles can never be beaten.

Reply:
I can remove it here as well - https://github.com/dcos-labs/ui-kit/blob/90280672f59c9af244a8c2e5838294c3bb5969a7/packages/shared/styles/styleUtils/modifiers/modifierUtils.ts#L35 .